.github/workflows/ci.yml

```diff
@@ -5,19 +5,20 @@ on:
     branches:
       - 'master'
   pull_request:
+  workflow_dispatch:
 
 jobs:
     testsuite:
+        name: "Test Suite"
         runs-on: ubuntu-18.04
         strategy:
             fail-fast: false
             matrix:
-                php-version: [ '7.3', '7.4', '8.1' ]
+                php-version: [ '7.4', '8.0', '8.1' ]
                 db-type: [ sqlite, mysql, pgsql, agnostic ]
                 symfony-version: [ '4-min', '4-max', '5-min', '5-max' ]
                 exclude:
-                  - php-version: '8.1'
```

Comment:
Why removing 8.1 here?

Reply:
Because we're not excluding PHP 8.1 anymore?

Reply:
Thats a matrix
So you are now excluding all 4-min runs, not just the one with 8.1